Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN rpm -i /app/tmp/ilorest-3.0.1-7.x86_64.rpm; \
     chmod -Rv g-rwx /root/.ssh; chmod -Rv o-rwx /root/.ssh; \
     rm -rf /app/tmp; \
     cd /usr/local/bin; \
-    curl https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz | tar xvzf -; \
+    curl https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.7/openshift-client-linux.tar.gz | tar xvzf -; \
```

Comment:
Most version numbers are pinned in [versions.ini](https://github.com/project-faros/cluster-manager/blob/master/app/versions.ini). It was actually an oversight that this wasn't pinned. Dynamically pinning it with the versions.ini file would be better, but this will work for now.